config/index.js

```diff
@@ -1,3 +1,8 @@
 module.exports = {
-  rDeployDirectory: '.rdeploy'
+  rDeployDirectory: '.rdeploy',
+  divisions: {
```

Comment:
Does this specify the different divisions? If so would we be able to specify multiple eligible divisions?